src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ForceLoadActiveConfiguredProjects.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright(c) Microsoft.All Rights Reserved.Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    /// <summary>
+    ///     Force loads the active <see cref="ConfiguredProject"/> objects so that any configured project-level 
+    ///     services, such as evaluation and build services, are started.
+    /// </summary>
+    internal class ForceLoadActiveConfiguredProjects : OnceInitializedOnceDisposed
+    {
+        private readonly IActiveConfigurationGroupService _activeConfigurationGroupService;
+        private readonly UnconfiguredProject _project;
+        private IDisposable _subscription;
+
+        [ImportingConstructor]
+        public ForceLoadActiveConfiguredProjects(UnconfiguredProject project, IActiveConfigurationGroupService activeConfigurationGroupService)
+        {
+            _project = project;
+            _activeConfigurationGroupService = activeConfigurationGroupService;
+        }
+
+        [ProjectAutoLoad(ProjectLoadCheckpoint.ProjectInitialCapabilitiesEstablished)]
```

Comment:
@lifengl This looks like the right time to do this - just after we've flowed the configured project capabilities -> unconfigured project. I could go this or  `AfterLoadInitialConfiguration`